roboticsrov/stream/stream_constants.py

```diff
@@ -0,0 +1,3 @@
+STREAM_RESOLUTION = '640x480'
+STREAM_FRAMERATE = 5
```

Comment:
That was just a temp value, it seems to work smoothly up to 30 though more testing is needed.